<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Discs</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="4,4,1" columnminimumwidth="0,0,0">
    <item row="0" column="0">
     <widget class="SimulationWidget" name="simulationWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
       <item>
        <widget class="PlotWidget" name="plotWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PlotControlWidget" name="plotControlWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="SimulationControlWidget" name="simulationControlWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1212</width>
     <height>24</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="loadSettingsFromJsonAction"/>
    <addaction name="saveSettingsAsJsonAction"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="saveSettingsAsJsonAction">
   <property name="text">
    <string>Save settings as...</string>
   </property>
  </action>
  <action name="loadSettingsFromJsonAction">
   <property name="text">
    <string>Open JSON settings...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimulationWidget</class>
   <extends>QWidget</extends>
   <header>widgets/SimulationWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>widgets/PlotWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotControlWidget</class>
   <extends>QWidget</extends>
   <header>widgets/PlotControlWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimulationControlWidget</class>
   <extends>QWidget</extends>
   <header>widgets/SimulationControlWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
